src/focus-ring.js

```diff
@@ -1,17 +1,12 @@
-import classList from 'dom-classlist';
 
 /**
  * https://github.com/WICG/focus-ring
  */
 function init() {
+  var hadKeyboardEvent = true;
   var elWithFocusRing;
 
   var inputTypesWhitelist = {
-    'radio': true,
```

Comment:
I don't think so. That would cause focus-ring to be applied on mouse click because these get checked during the focus event. The reverted version added them because it was checking only on keyUp.